en/docs/learn/pending-account-status.md

```diff
@@ -36,8 +36,9 @@ This account status can have the following values.
     2. PENDING_AP
     3. PENDING_EV
     4. PENDING_SR
-    5. DISABLED
-    6. UNLOCKED
+    5. PENDING_LR
```

Comment:
can we move this Pending state after PENDING_SR? Then all Pending states will be grouped together